pennylane/ops/functions/generator.py

```diff
@@ -135,10 +135,10 @@ def generator(op: qml.operation.Operator, format="prefactor"):
 
         * ``"observable"``: Return the generator as a single observable as directly defined
           by ``op``. Returned generators may be any type of observable, including
-          :class:`~.Hermitian`, :class:`~.SparseHamiltonian`, or :class:`~.Hamiltonian`.
+          :class:`~.Hermitian`, :class:`~.SparseHamiltonian`, or :class:`~.ops.LinearCombination`.
```

Comment:
maybe this doesn't make so much sense since it's not an observable anymore, but if we are removing Hamiltonian and potentially deprecating Observable, I don't know how much these categories will make sense anymore in general

Reply:
Maybe we can just start interpretting "observable" as "probably hermitian operator".  

But yes, I'm not quite sure if these categories make much sense anymore.